book.rb

```diff
@@ -0,0 +1,15 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- [x] Kindly I recommend you to set `*Getters and Setters for @rentals` instead of setting only  `attr_reader for @rentals` because when you set up `many-to-many` (also `has-many-through`) relationship between `Person` and `Book` using the intermediate class `Rental`, A book should have many rentals so in `attr_accessor` provides both read and write access to an instance variable, while attr_reader only provides read access. So kindly Adjust like the below sample.
```

attr_accessor :title, :author, :rentals

```